Source/JavaScriptCore/jit/GPRInfo.h

```diff
@@ -741,10 +779,10 @@ class GPRInfo {
         return registerForIndex[index];
     }
 
-    static GPRReg toArgumentRegister(unsigned index)
+    static constexpr GPRReg toArgumentRegister(unsigned index)
```

Comment:
One other set of changes that might've been a good candidate for landing separately is the migration of various functions to use constexpr. Not saying you need to split it off but in the future doing so will make it easier to review (less ambiguity about what constitutes the "meat" of the patch)